nilearn/plotting/surf_plotting.py

```diff
@@ -422,6 +426,248 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
+def _check_hemispheres(hemispheres):
+    """Checks whether the hemisphere in plot_img_on_surf is correct.
+
+    hemisphere: list
+        Any combination of 'left' and 'right'
+    """
+    if not isinstance(hemispheres, list):
```

Comment:
I suggest that we remove this check, someone might pass for example a tuple. let's not accept a string as input, ie for just left the user needs to pass `["left"]`, not `"left"`

Reply:
Done.